test/unit/messaging/submission-farmer.json

```diff
@@ -2,8 +2,16 @@
   "confirmationId": "WM-CE7-201",
   "abstractionLicence": "Expected to have by 31st December 2021",
   "agentAddressDetails": null,
-  "agentContactDetails": null,
-  "agentDetails": null,
+  "agentContactDetails": {
```

Comment:
This JSON file is to mimic the data sent from a farmer submission, so we don't want the `agentContactDetails` or `agentDetails` in here

Reply:
but this fix the broken tests